src/topkg_build.ml

```diff
@@ -33,11 +38,16 @@ let with_dir b dir = { b with dir }
 let nop = fun _ -> Ok ()
 
 let cmd c os files =
-  Topkg_os.Cmd.run @@ Topkg_cmd.(build_cmd c os %% of_list files)
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:false c os %% v "-just-plugin") >>= fun () ->
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:true c os %% of_list files)
```

Comment:
@gasche I had no idea `-plugin-option` existed, and I'm mildly confused as to its semantics. Does `-plugin-option x` mean "run ocamlbuild -just-plugin without x, then run ocamlbuild with x"? Or does it have different semantics depending on whether myocamlbuild.ml exists?